analyses/fusion_filtering/README.md

```diff
@@ -0,0 +1,47 @@
+## Fusion prioritization
+
+**Module author:** Krutika Gaonkar ([@kgaonkar6](https://github.com/kgaonkar6))
+Jaclyn Taroni ([@jaclyn-taroni] (https://github.com/jaclyn-taroni))
+Jo Lynne([@jharenza](https://github.com/jharenza))
+Komal Rathi ([@komalsrathi] (https://github.com/komalsrathi))
+
+
+This analysis will perform artifact filtering and annotation on fusion calls from STARfusion and Arriba fusion callers to prioritize putative oncogenic fusions. 
+Briefly, we considered all inframe and frameshift fusion calls with a minimum of 1 junction reads and at least one gene partner expressed (TPM > 1) to be true calls. 
+If a fusion call had large number of spanning fragment reads compared to junction reads (spanning fragment minus junction read greater than ten), we removed these calls as potential false positives. 
+We prioritized a union of fusion calls as true calls if the fused genes were detected by both callers OR the same fusion was recurrent within a broad_histology (>2 samples) OR the fusion was specific to the broad_histology. 
+If either 5' or 3' genes fused to more than five different genes within a sample, we removed these calls as potential false positives. 
+We annotated putative driver fusions and prioritized fusions based on partners containing known kinases, oncogenes, tumor suppressors, curated transcription factors, cosmic conses gene list or TCGA fusions.
```

Comment:
```suggestion
We annotated putative driver fusions and prioritized fusions when at least one fused gene was a known kinase, oncogene, tumor suppressor, curated transcription factor, on the cosmic consensus gene list.
We also annotated fusions between pairs of genes that were observed in TCGA.
```
I think this means the same thing as what you wrote but I was really not sure on the TCGA bit. I didn't know if one member had to be in the TCGA fusion or if both did. This version specifies that both do.

Reply:
you are right, the fusion between the pair of genes needs to be in TCGA